### DISCUSSION TITLE
Support for Jieshuo Screen Reader as an available screen reader for TT Droid

### DISCUSSION THREAD

**Discussion by CiprianDinca9** [BearWare/TeamTalk5#2046] [Ideas]
### Description

Jieshuo Screen Reader, also known as Commentary Screen reader, is the newer screen reading software for Android. When we check that events go through a screen reader, Jieshuo doesn't speak, which means that it's not supported. Here's the releases page if you want to test.
Thanks and advance!
https://github.com/nirenr/jieshuo/releases

### Application

- [ ] qtTeamTalk
- [X] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkServer

**Comment by beqabeqa473** [comment]
Please be more specific.

describe problem in details.

**Comment by Ramon-Salazar** [comment]
I think it's unneaded, all of the events are spoken through TTS, which is better since you can set an independend voice, not the voice used by the screenreader
